### ISSUE TITLE
New version of 0.17.2 causes Files and folders not to refresh and "hang" on deletion

### ISSUE THREAD

**Issue by frankbootmaker** [owncloud/music#801]
Hi, 

I have NC 20.0.1 and also an NC 19.0.4 and after installing the latest 0.17.2 Music app version:

- uploaded files did not automatically show up in the folder list but only after manual refresh
- deleting files was showing a "processing" icon but until manual refresh
- the empty function of the trash does not respond. Individually the files can be deleted but they will vanish only after manual refresh

I tested the behaviour on both versions.
If I disable the app the issue stops, therefore I believe there is a connection between the latest Music app version and the error described.

Unfortunately there is no log being created but the behaviour is there...

**Comment by paulijar** [comment]
This is a duplicate of #798, although with a more descriptive title. Shall be fixed soon.
